changelog.d/881.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Fixed a bug in errors raised in ``_parser.py`` in which the ``ValueError``
```

Comment:
This should not mention `_parser.py`, which is a private module, and should be updated to reflect the fact that parsing errors will now raise `ParserError`, which is a subclass of `ValueError`, and which has a cleaner string representation when passed a `msg`, `timestr` tuple.